config/environments/production.rb

```diff
@@ -43,8 +44,12 @@
   # Don't log any deprecations.
   config.active_support.report_deprecations = false
 
-  # Replace the default in-process memory cache store with a durable alternative.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :redis_cache_store,
+                       {
+                         url: ENV.fetch("REDIS_URL"),
+                         ssl_params: { verify_mode: OpenSSL::SSL::VERIFY_NONE },
+                         expires_in: 24.hours,
+                       }
```

Comment:
In any other context, I'd think assignment with a comma was a mistake, but this is indeed the recommended way of setting it up!